server/config.js

```diff
@@ -107,6 +107,26 @@ var conf = convict({
     env: 'REDIS_DATABASE',
     format: 'int'
   },
+  db_reindex_timeout: {
+    doc: 'Number of milliseconds for etl to fetch from postgres + bulk insert into elasticsearch',
+    format: 'int',
+    default: 30000
+  },
+  db_reindex_batchSize: {
+    doc: 'Batch size for elasticsearch reindexing',
+    default: 100,
+    format: 'int'
+  },
+  db_reindex_maxRetries: {
+    doc: 'Number of times to retry an individual ETL chunk before giving up completely',
+    default: 5,
+    format: 'int'
+  },
+  db_reindex_retryWait: {
+    doc: 'Base wait period before retrying after a failure. There is a linear multiplier in the code.',
+    default: 10000,
+    format: 'int'
```

Comment:
Sure, I can take another stab at creating a PR for that.
I think we reverted it out last time because it could be a duration in the convict file but not in the JSON files (so we'd could do "10 seconds" in the convict.js file but the JSON files needed to use 10000). I'll try retesting that locally with the latest Convict.